src/engine/main/CMakeLists.txt

```diff
@@ -175,6 +175,8 @@ target_link_libraries(engine_ser_exe
     ${CMAKE_THREAD_LIBS}
     ${ZLIB_LIBRARY}
 )
+# For linuxppc64le
+TARGET_LINK_LIBRARIES(engine_ser_exe ${DL_LIB})
 
```

Comment:
Why not just add DL_LIB to the targe_link_libraries list above?

Reply:
Fixed